R/latent_lognormal.R

```diff
@@ -22,7 +23,7 @@ posterior_predict_latent_lognormal <- function(i, prep, ...) { # nolint: object_
     # while loop to impose the truncation
     while (d_censored > obs_t) {
       p_latent <- runif(1, 0, 1) * pwindow
```

Comment:
ibid runif in stats.

Reply:
https://github.com/epinowcast/epidist/pull/329/commits/0dd2f6ce9ec26a8f4ebd9f54115a44e5f3baea7b